src/lib/schematics/update/update-schematic.md

```diff
@@ -0,0 +1,169 @@
+# Update schematic
```

Comment:
`# Creating ng-update schematics`

Reply:
I'd prefer just `ng-update schematics` because it's not only about creating a update schematic.

I'd consider the whole update tool as _one_ schematic (the `update schematic`). And breaking changes will be covered by the `update/migration data`.